extensions/typescript-language-features/package.nls.json

```diff
@@ -40,7 +40,8 @@
 	"format.semicolons.ignore": "Don't insert or remove any semicolons.",
 	"format.semicolons.insert": "Insert semicolons at statement ends.",
 	"format.semicolons.remove": "Remove unnecessary semicolons.",
-	"format.indentSwitchCase": "Indent case clauses in switch statements. Requires using TypeScript 5.1+ in the workspace.",
+	"typescript.format.indentSwitchCase": "Indent case clauses in switch statements in TypeScript Files. Requires using TypeScript 5.1+ in the workspace.",
```

Comment:
Most of the formatting settings can use a single string for both javascript and typescript. See the settings above for `format.semicolons.remove` for example